e2e/cypress/src/cypress.test.ts

```diff
@@ -140,10 +140,13 @@ describe('env vars', () => {
     const run3 = runCLI(`e2e ${myapp}-e2e --no-watch`);
     expect(run3).toContain('All specs passed!');
 
-    expect(await killPorts(4200)).toBeTruthy();
+    expect(await killPort(4200)).toBeTruthy();
   }, 1000000);
 
-  it('should run e2e in parallel', () => {
+  it('should run e2e in parallel', async () => {
+    // ensure ports are free before running tests
+    await killPort(4200);
+
```

Comment:
interesting, because the whole point of this feature is that it can handle ports already taken, I guess makes sense to have a known clean slate, but test _ideally_ should not have been failing from that.